distributed/worker.py

```diff
@@ -621,7 +622,7 @@ def __init__(
             ("executing", "released"): self.transition_executing_released,
             ("executing", "rescheduled"): self.transition_executing_rescheduled,
             ("fetch", "flight"): self.transition_fetch_flight,
-            ("fetch", "missing"): self.transition_fetch_missing,
+            ("fetch", "missing"): self.transition_generic_to_missing,
```

Comment:
nit: all other generic transitions are called ``transition_generic_<finish>``; we should have consistency. I like this one way more, as it's more explicit.

Reply:
however, this transition is only used here. So it has no reason to be generic?

Reply:
I renamed to `transtition_generic_missing`. The generic part is that it doesn't depend on the input state